promscale/downsampling/recording.md

```diff
@@ -0,0 +1,65 @@
+# Recording rules in Promscale
+Promscale supports Prometheus recording rules. These rules
+are used to calculate frequently used or computationally expensive queries
+before they are required. A recording rule is a PromQL expression that
+Prometheus evaluates at a predefined frequency to generate a new metric series.
+The new metric series is stored in Promscale, and you can query it in the same
+way as any other Prometheus metric. In Promscale, you can set the recording 
+rules similar to Prometheus rules configuration.
+
+<procedure>
+
+## Set recording rules
+
+Promscale recording rules are built on top of Prometheus recording rules capabilities.
+Recording rules are written in a `YAML` file and specified in the Promscale configuration
+file.
+* `recording rules` are used for computationally expensive expressions and save
+their result as a new set of time series data. These help in visualisation of
+data over a period of time. 
```

Comment:
I don't understand what you mean here? Do you mean over a long period of time?

Reply:
yup, as the window for data querying grows. Executing computational queries at every data point makes things even worse. So recording rules compute at frequency set and over a long period of time, this will have easy querying as we do not need computational execution.